express/scripts/scripts.js

```diff
@@ -11,6 +11,7 @@
  */
 
 import { setLibs, decorateArea, listenMiloEvents } from './utils.js';
+import { autoUpdateContent } from "./content-replace.js"
```

Comment:
This should be a dynamic import. By adding your static import here you're blocking the whole script  from loading until content-replace.js also loads.